comp/remote-config/rcclient/rcclient.go

```diff
@@ -86,11 +87,26 @@ func newRemoteConfigClient(deps dependencies) (provides, error) {
 		return provides{}, err
 	}
 
+	var clientHA *client.Client
+	if config.Datadog.GetBool("ha.enabled") {
+		clientHA, err = client.NewUnverifiedHAGRPCClient(
+			ipcAddress,
+			config.GetIPCPort(),
+			security.FetchAuthToken,
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 Warning:** ‏`FetchAuthToken` now needs a `configModel.Reader` parameter, typing is wrong

Reply:
Rebased! Thanks for pointing this out.